pipelines/common/src/test/java/com/google/fhir/analytics/FhirStoreUtilTest.java

```diff
@@ -78,11 +78,14 @@ public void setup() throws IOException {
 
     patient = (Patient) patientBundle.getEntryFirstRep().getResource();
 
-    when(clientFactory.newGenericClient(sinkUrl)).thenReturn(client);
-    when(client.update().resource(patient).withId(patient.getId()).encodedJson()).thenReturn(iexec);
-    when(client.transaction().withBundle(ArgumentMatchers.any(Bundle.class)).execute())
+    lenient().when(clientFactory.newGenericClient(sinkUrl)).thenReturn(client);
```

Comment:
Avoid using `lenient()` in all cases and use it only when required, refer here for [usage](https://javadoc.io/doc/org.mockito/mockito-core/latest/org/mockito/Mockito.html#lenient())